### ISSUE TITLE
Count of returned items in a query?

### ISSUE THREAD

**Issue by koosham** [inukshuk/jekyll-scholar#186]
I was wondering if there is any "clean" way to get the count of bib items returned by a query. It will be useful in multiple ways:
- Conditional generation of headers/containers for publications (if the count is greater than zero)
- Printing the publication count

I'm going to implement this by counting the number of `<li>` tags using JavaScript, but I thought it can be a good improvement for this package.

**Comment by inukshuk** [comment]
Well, for the latter we could add a separate tag (or option to the bibliography tag) which just prints the number of items. I'm not sure about the first one, though -- how would that look in liquid?

We could also put the number of items into a data-attribute of the list; but that's hardly cleaner than just counting the `<li>` in the list.

**Comment by koosham** [comment]
You're right. I don't see any way liquid would be able to handle conditional statement, since you are generating a final string ready to be displayed, and not a list variable. But still, publication count will be a good feature and people in academia would love it, specifically, if their publication count is a large number :smile:
